chromium_src/chrome/browser/history/history_utils_unittest.cc

```diff
@@ -0,0 +1,12 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "chrome/browser/history/history_utils.h"
+#include "testing/gtest/include/gtest/gtest.h"
+#include "url/gurl.h"
+
+TEST(HistoryUtilsTest, BraveUISchemeTest) {
+  EXPECT_FALSE(CanAddURLToHistory(GURL("brave://sync/")));
```

Comment:
can we add some urls that should be added to history? Want to make sure that someone doesn't accidentally break this to return `false` for everything

Reply:
also some urls that should not be added based on the ChromiumImpl method?

Reply:
I think we don't need to to check many urls because `CanAddURLToHistory()` just checks scheme.

Reply:
Ah, I see what you mean. will add more urls

Reply:
Done.